packages/components/src/components/trace/Trace.vue

```diff
@@ -48,6 +49,10 @@ export default {
       type: Object,
       default: null,
     },
+    highlightedEvents: {
+      type: Array,
+      default: () => [],
```

Comment:
This should be a `Set`, otherwise this may lead to an `O(n^2)` operation as each event has to iterate the array to check if it must render a highlight. Everywhere we're drilling this prop should use a `Set` as well.

Reply:
Done https://github.com/applandinc/appmap-js/pull/496/commits/7faf9ed463eb9aae887b92f8fb2fa2db1d7f62e8